tests/system/providers/google/cloud/vertex_ai/example_vertex_ai_model_service.py

```diff
@@ -87,7 +87,9 @@
 
 CONTAINER_URI = "gcr.io/cloud-aiplatform/training/tf-cpu.2-2:latest"
 
-LOCAL_TRAINING_SCRIPT_PATH = "california_housing_training_script.py"
+LOCAL_TRAINING_SCRIPT_PATH = os.environ.get(
+    "VERTEX_AI_LOCAL_TRAINING_SCRIPT_PATH", "california_housing_training_script.py"
```

Comment:
Just curious, where does this variable come from?

Reply:
@moiseenkov from OS environment variable. On local users env or breeze env users can use the default value. For our CI envs I will specify this env variable with this value `gcs/data/california_housing_training_script.py`. I have added this change, because for Composer, our CI or any other distributed envs. The task can start on any worker and this file should be on each worker. In Composer env `gcs/data/` folder shares between workers and for fixing run for current system tests in our CI we need to upload  `california_housing_training_script.py` in this folder. But because we can't hardcoded this value in the community I decided to add ENV variable which we will use in our CI with the needed value.

Reply:
What about the case when we run this system test in composer manually (without CI)? In this case LOCAL_TRAINING_SCRIPT_PATH isn't set by default, and we might need some time to debug it. What about adding a comment explaining what the path should look like in the case of a Composer?

Reply:
@moiseenkov I have added the comment to this variable